binaries/apiserver/main.go

```diff
@@ -32,7 +32,7 @@ func main() {
 	level, err := log.ParseLevel(*logLevelFlag)
 	if err != nil {
 		log.Error(err)
-		return
+		level = log.InfoLevel
```

Comment:
Why do we want to continue starting up if there's an error parsing the level? Startup is the perfect time to bomb due to errors.